PIL/Image.py

```diff
@@ -1092,12 +1147,14 @@ def draft(self, mode, size):
         pass
 
     def _expand(self, xmargin, ymargin=None):
+        # type: (int, Optional[int]) -> Image
         if ymargin is None:
             ymargin = xmargin
         self.load()
         return self._new(self.im.expand(xmargin, ymargin, 0))
 
     def filter(self, filter):
+        # type: (ImageFilter.Filter) -> Image
```

Comment:
Can be a callable returning a filter as well. (ducktyped at that, that implements filter(ImagingCore))